src/main/webapp/mobile/receptionistapptstyle.css

```
@@ -159,16 +173,17 @@ a.encounterBtn, a.masterBtn {
-    top: 8px;
+    top: -3px;
     color: #FFFFFF;
     text-shadow: rgba(0, 0, 0, 0.6) 0px -1px 0;
     text-overflow: ellipsis;
     text-decoration: none;
     text-align: center;
     white-space: nowrap;
     z-index: 1;
-    -webkit-border-image: url(../mobile/toolButton.png) 0 5 0 5;
-    -moz-border-image: url(../mobile/toolButton.png) 0 5 0 5;
+    border: 0 solid transparent;
```

Stylelint (17.12.0)

[error] Overridden property "border-width" by shorthand "border" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)